### PR TITLE
add docker file

### PR DESCRIPTION
**Pull request by dsk7**

### CODE REVIEW

File: docker/scripts/entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+mkdir -p /srv/montag/books/{file_store,import_watch}
+mkdir -p /srv/montag/metadata/{db,config,logs}
```
Comment (WeizenJoe) [inline docker/scripts/entrypoint.sh:4]: Readme.MD gives this example:
`  -v /var/lib/montag/data:/srv/montag/data` 
Is it possible that line 4 here should read 'mkdir -p /srv/montag/data/{db,config,logs}' ?

Comment (WeizenJoe) [reply docker/scripts/entrypoint.sh:4]: Alternatively 'metadata' was intended and the readme should be fixed, it seems 'metadata' is used in several places and appears to be the better term.

File: docker/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+Pass the following commands to docker run
+
+`  -v /var/lib/montag/data:/srv/montag/data` 
+
+This puts all metadata to /var/lib/montag/data
+
+
+`  -v /large_volume/books:/srv/montag/books`
+
+This puts all books to /var/lib/montag/books
```
[the remark this answers is not in the document]
Comment (dsk7) [reply docker/README.md:27]: fixed

File: docker/scripts/entrypoint.sh (hunk 1 above)
Comment (dsk7) [reply docker/scripts/entrypoint.sh:4]: changed to metadata in readme

**Pull request merged**